{"name": "test_02", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = hasattr('hello', 'hello')", "trace": "def test_02():\n        print (\"正在执行test2\")\n        x = \"hello\"\n>       assert hasattr(x,\"hello\")\nE       AssertionError: assert False\nE        +  where False = hasattr('hello', 'hello')\n\ntest_function.py:21: AssertionError"}, "attachments": [{"name": "log", "source": "7050fe06-546c-4ffd-a18c-45bc06a56de8-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "8411efbf-9883-4925-ba0d-a6a4b96611d6-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "bb3e5a2d-634a-43b2-ba93-2c69cd58fc81-attachment.txt", "type": "text/plain"}], "start": 1564493502091, "stop": 1564493502091, "uuid": "9965e06e-43de-44f8-a5a8-39714d50d231", "historyId": "009b09508e788ab0fa805f42789680bc", "fullName": "test_function#test_02", "labels": [{"name": "suite", "value": "test_function"}, {"name": "host", "value": "anxiaodongdeMacBook-Pro.local"}, {"name": "thread", "value": "84097-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_function"}]}